apostrophecms/apostrophe
# refactor component logic into separate files

## Summary
Refactor schema component logic to support styling overrides [PRO-4651](https://linear.app/apostrophecms/issue/PRO-4651/update-refactored-components-in-the-holon-project) [HOL-2099](https://holonsolutions.atlassian.net/browse/HOL-2099?focusedCommentId=38337)

## What are the specific steps to test this change?
1. `npm link` this pull in Holon
2. in Holon `npm run dev`
3. Navigate to a site
4. Click appBuilder in nav bar > Add category
5. The modal loads with content and without errors in the console
6. All modal components should render as normal without error in the console

## What kind of change does this PR introduce?

## Make sure the PR fulfills these requirements:
